### DISCUSSION TITLE
ruckus/ZDAddLicenses

### DISCUSSION THREAD

**Discussion by giscus** [ms264556/Hackery#15] [Comments]
# ruckus/ZDAddLicenses

Hackery

https://ms264556.net/ruckus/ZDAddLicenses

**Comment by smfilip** [comment]
Hi, I followed this tutorial for ZD1200, but now I am stuck on version 9.9.1.0 build 52, and my APs can't go that low (R600). When I try to upgrade firmware to ANY higher version, I only get notification that support is expired... Did I just bricked my ZD? Please help, if there is help :(

**Reply by ms264556** [reply]
This page is old. I didn't think it was linked anymore & it should've been deleted.

Please contact me privately at [ServeTheHome](https://forums.servethehome.com/index.php?members/ms264556.37738/) or [Reddit](https://www.reddit.com/user/Famous-Fishing-1554/) and I'll sort you out with a fix.
